packages/account/src/Helpers/utils.ts

```diff
@@ -1,8 +1,10 @@
+import React from 'react';
```

Comment:
do we really need React import in the utils file?

Reply:
Its being used for type. I could destructure the React import object, but that not the style being used

Reply:
@likhith-deriv this is a .ts file, can you elaborate more